src/thor/isochrone_action.cc

```diff
@@ -31,18 +37,35 @@ std::string thor_worker_t::isochrones(Api& request) {
   // Cost (including penalties) is used when adding to the adjacency list but the elapsed
   // time in seconds is used when terminating the search. The + 10 minutes adds a buffer for edges
   // where there has been a higher cost that might still be marked in the isochrone
-  auto grid = (costing == "multimodal" || costing == "transit")
-                  ? isochrone_gen.ComputeMultiModal(*options.mutable_locations(),
-                                                    contours.back() + 10, *reader, mode_costing, mode)
-                  : isochrone_gen.Compute(*options.mutable_locations(), contours.back() + 10, *reader,
-                                          mode_costing, mode);
-
-  // turn it into geojson
-  auto isolines =
-      grid->GenerateContours(contours, options.polygons(), options.denoise(), options.generalize());
-
-  return tyr::serializeIsochrones<PointLL>(request, isolines, options.polygons(), colors,
-                                           options.show_locations());
+  auto grid =
+      (costing == "multimodal" || costing == "transit")
+          ? isochrone_gen.ComputeMultiModal(*options.mutable_locations(),
+                                            contours_time.front() + 10.0f,
+                                            contours_distance.front() + 10.0f, *reader, mode_costing,
+                                            mode)
+          : isochrone_gen.Compute(*options.mutable_locations(), contours_time.front() + 10.0f,
+                                  contours_distance.front() + 10.0f, *reader, mode_costing, mode);
```

Comment:
Actually I'm not so happy with adding 10 km here.. For time it's not so terrible to add 10 mins as we modulate the grid size with the mode speed, so non-motor vehicles get smaller grids than motorized vehicles. but adding 10 km means a 1 km pedestrian iso gets an 11 km grid assigned, which makes the grid bounds much larger and increases the blockiness.. might be fine for now, but maybe the whole grid definition should change a little eventually?